candid/governance.did

```diff
@@ -1,10 +1,336 @@
+  JoinCommunityFund : record {};
+  SetDissolveTimestamp : SetDissolveTimestamp;
+};
+type Proposal = record {
+  url : text;
+  title : opt text;
+  action : opt Action;
+  summary : text;
+};
+type ProposalData = record {
+  id : opt NeuronId;
+  failure_reason : opt GovernanceError;
+  ballots : vec record { nat64; Ballot };
+  proposal_timestamp_seconds : nat64;
+  reward_event_round : nat64;
+  failed_timestamp_seconds : nat64;
+  reject_cost_e8s : nat64;
+  latest_tally : opt Tally;
+  decided_timestamp_seconds : nat64;
+  proposal : opt Proposal;
+  proposer : opt NeuronId;
+  wait_for_quiet_state : opt WaitForQuietState;
+  executed_timestamp_seconds : nat64;
+};
+type ProposalInfo = record {
+  id : opt NeuronId;
+  status : int32;
+  topic : int32;
+  failure_reason : opt GovernanceError;
+  ballots : vec record { nat64; Ballot };
+  proposal_timestamp_seconds : nat64;
+  reward_event_round : nat64;
+  deadline_timestamp_seconds : opt nat64;
+  failed_timestamp_seconds : nat64;
+  reject_cost_e8s : nat64;
+  latest_tally : opt Tally;
+  reward_status : int32;
+  decided_timestamp_seconds : nat64;
+  proposal : opt Proposal;
+  proposer : opt NeuronId;
+  executed_timestamp_seconds : nat64;
+};
+type RegisterVote = record { vote : int32; proposal : opt NeuronId };
+type RemoveHotKey = record { hot_key_to_remove : opt principal };
+type Result = variant { Ok; Err : GovernanceError };
+type Result_1 = variant { Error : GovernanceError; NeuronId : NeuronId };
+type Result_2 = variant { Ok : Neuron; Err : GovernanceError };
+type Result_3 = variant { Ok : RewardNodeProviders; Err : GovernanceError };
+type Result_4 = variant { Ok : NeuronInfo; Err : GovernanceError };
+type RewardEvent = record {
+  day_after_genesis : nat64;
+  actual_timestamp_seconds : nat64;
+  distributed_e8s_equivalent : nat64;
+  settled_proposals : vec NeuronId;
+};
+type RewardMode = variant {
+  RewardToNeuron : RewardToNeuron;
+  RewardToAccount : RewardToAccount;
+};
+type RewardNodeProvider = record {
+  node_provider : opt NodeProvider;
+  reward_mode : opt RewardMode;
+  amount_e8s : nat64;
+};
+type RewardNodeProviders = record { rewards : vec RewardNodeProvider };
+type RewardToAccount = record { to_account : opt AccountIdentifier };
+type RewardToNeuron = record { dissolve_delay_seconds : nat64 };
+type SetDefaultFollowees = record {
+  default_followees : vec record { int32; Followees };
+};
+type SetDissolveTimestamp = record { dissolve_timestamp_seconds : nat64 };
+type Spawn = record { new_controller : opt principal; nonce : opt nat64 };
+type SpawnResponse = record { created_neuron_id : opt NeuronId };
+type Split = record { amount_e8s : nat64 };
+type Tally = record {
+  no : nat64;
+  yes : nat64;
+  total : nat64;
+  timestamp_seconds : nat64;
+};
+type UpdateNodeProvider = record { reward_account : opt AccountIdentifier };
+type WaitForQuietState = record { current_deadline_timestamp_seconds : nat64 };
+service : (Governance) -> {
```

Comment:
Note:  Where previously we had `service {` we now have an argument.  This does not, however, seem to have had any effect on the generated js so I believe that this is safe to ignore.

Reply:
Yes, this has no affect at all on consumers.